sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -2194,6 +2194,20 @@ val SHUFFLE_COMPRESSION_LZ4_CHUNK_SIZE = conf("spark.rapids.shuffle.compression.
       .integerConf
       .createWithDefault(1024)
 
+  val FORCE_ONE_FILE_PER_PARITION =
```

Comment:
If we're also requiring the per-file reader to be enabled, then another way to accomplish this is to disable splitting on DeltaParquetFileFormat, since per-file will force file-at-a-time in a task and unsplittable input will prevent files from being shared across tasks.  Not sure if there's a clean way to control the splittability of DeltaParquetFileFormat when merging, but if so, would be a cleaner solution that this hidden config.